src/d3/scatterplot.d3.ts

```diff
@@ -157,3 +168,23 @@ export class Scatterplot{
     });
   }
 }
+
+AFRAME.registerComponent('rotate_cards', {
+  tick() {
+    this.el.object3D.rotation.set(
+       document.querySelector('[camera]').object3D.rotation.x,
+       document.querySelector('[camera]').object3D.rotation.y,
+       document.querySelector('[camera]').object3D.rotation.z,
+    );
+  }
+});
+
+AFRAME.registerComponent('show_cards', {
+  tick() {
+    if (this.el.object3D.position.distanceTo(document.querySelector('[camera]').object3D.position) < DATA_PT_RADIUS * 40){
```

Comment:
Since we moved forward with the design that only shows the card if it is being hovered over with the mouse/laser cursor, then this component is no longer necessary